google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryResultRowFormat.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+import com.google.auto.value.AutoValue;
+import java.io.Serializable;
+
+/** Base class for Google BigQuery query result formats. */
+@AutoValue
+public abstract class QueryResultRowFormat implements Serializable {
```

Comment:
You have both QueryResultRowFormat here, and ResponseRowFormat further down.  Unclear how they're related.

Reply:
Removed `QueryResultRowFormat` class